specreduce/background.py

```diff
@@ -201,6 +210,7 @@ def one_sided(cls, image, trace_object, separation, **kwargs):
         crossdisp_axis : int
```

Comment:
is this parameter being used or is the assumption hard-coded as implied in the previous docstring?

Reply:
`cls` should be of type `Background`, so it follows the docstring's implication once the new instance is created at the end of the method.